rln/src/utils.rs

```diff
@@ -85,7 +85,7 @@ pub fn fr_to_bytes_be(input: &Fr) -> Vec<u8> {
 pub fn vec_fr_to_bytes_le(input: &[Fr]) -> Vec<u8> {
     let mut bytes: Vec<u8> = Vec::new();
     //We store the vector length
-    bytes.extend(input.len().to_le_bytes().to_vec());
+    bytes.extend(u64::try_from(input.len()).unwrap().to_le_bytes().to_vec());
```

Comment:
This was added because `len()` returns an `usize`, so the number of bytes returned by `to_le_bytes` will change depending on the architecture